### ISSUE TITLE
Doesn't work well with screen readers

### ISSUE THREAD

**Issue by hharnisc** [hharnisc/tabbie#24]
Using a screen reader does list off the add/remove buttons, but the list items never get focused. So it's not clear which tab group is in focus.
